static/styles/zulip.css

```diff
@@ -190,19 +190,10 @@ p.n-margin {
     top: 0px;
 }
 
-.app-main .column-left .left-sidebar,
 .app-main .column-right .right-sidebar {
     position: fixed;
     margin-top: 50px;
     z-index: 100;
-}
-
-.app-main .column-left .left-sidebar {
-    width: 242px;
-    padding-left: 0px;
-}
-
-.app-main .column-right .right-sidebar {
     padding-left: 15px;
     width: 235px;
 }
```

Comment:
Yeah I undid this part of the migration because I think it's a bit clearer to have the high-level layout stuff all together.